pkg/pmem-csi-driver/nodeserver.go

```diff
@@ -374,6 +413,7 @@ func (ns *nodeServer) NodeUnstageVolume(ctx context.Context, req *csi.NodeUnstag
 		return nil, err
 	}
 
+	ns.volFstype[req.VolumeId] = ""
```

Comment:
When exactly we clean/remove the cache cahced map(s) elements?

Reply:
thanks, indeed I should use delete() instead of just overwriting with empty, added delete() calls in latest code.